src/modules/apiCall.js

```diff
@@ -0,0 +1,17 @@
+const dataJson = [''];
+const apiCall = async (homePageOption) => {
+  await fetch(`https://api.tvmaze.com/search/shows?q=${homePageOption[0]}`)
+    .then((data) => data.json())
+    .then((data) => {
+      dataJson.length = 0;
+      dataJson.push(data);
+    });
```

Comment:
- [ Optional] Please add validation to the comment section so the user doesn't add a very long comment or name 
![image](https://user-images.githubusercontent.com/74744787/211096261-08fb60e3-98e3-4ac9-936f-df230fe8f669.png)